packages/testing/src/consensus_testing/test_fixtures/fork_choice.py

```diff
@@ -254,7 +254,7 @@ def _build_block_from_spec(self, spec: BlockSpec, store: Store) -> SignedBlockWi
 
         # Create signed structure with placeholder signatures
         # One signature for proposer attestation + one for the block
-        signature_list = [Bytes4000.zero(), Bytes4000.zero()]
+        signature_list = [Bytes3100.zero(), Bytes3100.zero()]
```

Comment:
```suggestion
        signature_list = [Signature.zero(), Signature.zero()]
```
This will require importing Signature from subspecs.containers.